src/style.css

```
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
```

Add default font-weight
```suggestion
html {
  font-family: "Avenir", sans-serif;
  font-weight: 300;
}
```